src/client/AccountsClient.js

```diff
@@ -10,13 +10,13 @@ import AccountsCommon from '../common/AccountsCommon';
 import { loggingIn } from './module';
 import ui from './ui';
 import type { AccountsOptionsType } from '../common/AccountsCommon';
+import type UserObjectType from '../common/UserObjectType';
 
-export type UserObjectType = {username: ?string, email: ?string, id: ?string};
 const isValidUserObject = (user: UserObjectType) => has(user, 'user') || has(user, 'email') || has(user, 'id');
 
 export type UserCreationInputType = {
   username: ?string,
-  password: ?string,
+  password: string,
```

Comment:
I'm following the Meteor accounts documentation. On the client calling Accounts.createUser a password is required. On the server Accounts.createUser can be called without a password but the user wont be able to login until Accounts.setPassword is called. 

Accounts.createUser isn't used for oauth or other strategies, only for passwords. The other strategies will be handled by another method, probably Accounts.createUserFromService